generators/wp/templates/chisel-starter-theme/Chisel/Media.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Chisel;
+
+/**
+ * Class Media
+ * @package Chisel
+ *
+ * Default media settings for Chisel
+ */
+class Media {
+
+	public function __construct() {
+		// $this->addImageSizes();
+		// add_action( 'after_setup_theme', array( $this, 'defaultMediaSetting' ) );
+		// add_filter( 'image_size_names_choose', array( $this, 'customImageSizes' ) );
+		add_action( 'upload_mimes', array( $this, 'customMimeTypes') );
+		add_action( 'jpeg_quality', array( $this, 'customJpegQuality') );
+		add_filter( 'embed_oembed_html', array( $this, 'customOembedFilter' ), 10, 4);
+	}
+
+	/**
+	 * Add various image sizes
+	 */
+	public function addImageSizes() {
```

Comment:
This one is already introduced in https://github.com/xfiveco/generator-chisel/blob/master/generators/wp/templates/chisel-starter-theme/Chisel/WpExtensions.php#L44

We should get rid of this and leave it here.